src/grids.jl

```diff
@@ -25,7 +25,12 @@ end
 Construct a one-dimensional staggered `grid` on domain `x ∈ [x_start, x_end]` with topology
 `Tx`, with `nx` interior grid points, and `hx` halo points.
 """
-function Grid1D(Tx, nx, x_start, x_end; hx=0)
+function Grid1D(Tx, nx, x_start, x_end; hx=1)
+
+    if(hx == 0)
```

Comment:
```suggestion
    if hx == 0
```

@dhruvbhagtani you often don't put space between the end of a word and a parenthesis... Is that intentional?

Reply:
Haha, it's just the way I subconsciously do my coding.

Reply:
You also do it in your normal text write up. :)